packages/apps/human-app/frontend/src/pages/worker/jobs/components/avaible-jobs/available-jobs-table.tsx

```diff
@@ -0,0 +1,123 @@
+import type { MRT_ColumnDef } from 'material-react-table';
+import {
+  MaterialReactTable,
+  useMaterialReactTable,
+} from 'material-react-table';
+import { useQuery } from '@tanstack/react-query';
+import { t } from 'i18next';
+import { useEffect, useMemo, useState } from 'react';
+import { useTableQuery } from '@/components/ui/table/table-query-hook';
+import { SearchForm } from '@/pages/playground/table-example/table-search-form';
+import { Button } from '@/components/ui/button';
+import { Chip } from '@/components/ui/chip';
+import { useJobsFilterStore } from '@/hooks/use-jobs-filter-store';
+import { shortenEscrowAddress } from '../utils/shorten-escrow-address';
+import { parseNetworkName } from '../utils/parse-network-label';
+import type { AvailableJobs, JobsArray } from './available-jobs-table-service';
+import { getJobsTableData } from './available-jobs-table-service';
+
+const columns: MRT_ColumnDef<JobsArray>[] = [
+  {
+    accessorKey: 'escrowAddress',
+    header: 'Escrow address',
+    size: 100,
+    enableSorting: true,
+  },
+  {
+    accessorKey: 'network',
+    header: 'Network',
+    size: 100,
+    enableSorting: true,
+  },
+  {
+    accessorKey: 'jobTypeChips',
+    header: 'Job type',
+    size: 200,
+    enableSorting: true,
+  },
+  {
+    accessorKey: 'buttonColumn',
+    header: '',
+    size: 100,
+    enableSorting: true,
+  },
+];
+
+export function AvailableJobsTable() {
+  const { setFilterParams, filterParams } = useJobsFilterStore();
+
+  useEffect(() => {
+    setFilterParams({});
+  }, [setFilterParams]);
```

Comment:
should be call on component unmount. And should be call with initial state. Add comment that, there is filters clearing